waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -811,7 +792,18 @@ proc validateMessage*(rlnPeer: WakuRLNRelay, msg: WakuMessage,
   let
     contentTopicBytes = msg.contentTopic.toBytes
     input = concat(msg.payload, contentTopicBytes)
-    proofVerificationRes = rlnPeer.rlnInstance.proofVerify(input, msg.proof)
+  
```

Comment:
These lines were grouped together with line `806`  i.e.,
```
let proofVerificationRes = rlnPeer.rlnInstance.proofVerify(input, msg.proof)
```

Reply:
Addressed in [9f76dab](https://github.com/status-im/nwaku/pull/1158/commits/9f76dab6e712b951feb10f10af19f1700d9c6712)